include/linux/soundwire/sdw.h

```diff
@@ -789,6 +789,7 @@ struct sdw_master_ops {
  * struct sdw_bus - SoundWire bus
  * @dev: Master linux device
  * @link_id: Link id number, can be 0 to N, unique for each Master
+ * @id: Unique id for each bus
```

Comment:
maybe reword:

* @link_id: Link id number, can be 0 to N, unique for each Master in the controller scope.
 * @id: bus system-wide unique id

Reply:
updated.